Casks/g/green-go-control.rb

```diff
@@ -0,0 +1,31 @@
+cask "green-go-control" do
+  version "5.1.0"
+  sha256 "5e654801997ae166c0218b4e1fd0cafe011b4361714904dc2fcb7b56698dd72e"
+
+  url "https://downloads.greengoconnect.com/#{version}/macos/green-go-control.dmg",
+      verified: "downloads.greengoconnect.com/"
+  name "Green-GO Control"
+  desc "Configure and manage Green-GO intercom systems"
+  homepage "https://www.greengodigital.com/"
+
+  livecheck do
+    url "https://manual.greengoconnect.com/en/release-notes/software/"
+    regex(%r{href=.*?/downloads.greengoconnect.com/(\d+(?:\.\d+)*)/macos/green-go-control.dmg}i)
+    strategy :page_match
```

Comment:
```suggestion
    regex(%r{href=.*?/(\d+(?:\.\d+)+)/macos/green-go-control.dmg}i)
```

The strategy is `:page_match` by default so we don't need to set it.